_data/api-swagger.yaml

```diff
@@ -256,6 +256,20 @@ definitions:
                 - `*Emphasis*` / `_Emphasis_`
                 - `**Stong**` / `__Strong__`
                 - `1. Ordered` and `- Unordered` lists
+            Authentications:
+              description: |
+                List of authentications that the signer has to authenticate with.
```

Comment:
I don't think it's really clear to the reader what the difference is between an authentication and a verification, by just reading the two descriptions.